Index: dashboard.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from tkinter import *\r\n\r\nroot= Tk()\r\nroot.title(\"Python SQL Inventory\")\r\nroot.geometry(\"1270x668\")\r\nroot.config(background = \"#f8e9d2\")\r\nicon=PhotoImage(file=\"inventory.png\")\r\ntitlelabel=Label(root,text=\"Inventory Management Systems\",image=icon,compound=LEFT,font=(\"Arial\",30,\"bold\"),background=\"pink\",fg=\"white\")\r\ntitlelabel.place(x=0,y=0,relwidth=1)\r\n\r\n\r\nroot.mainloop()
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/dashboard.py b/dashboard.py
--- a/dashboard.py	(revision 23d2cf891331f912ec33ed74d21fee40a9f5931b)
+++ b/dashboard.py	(date 1759313310396)
@@ -7,6 +7,6 @@
 icon=PhotoImage(file="inventory.png")
 titlelabel=Label(root,text="Inventory Management Systems",image=icon,compound=LEFT,font=("Arial",30,"bold"),background="pink",fg="white")
 titlelabel.place(x=0,y=0,relwidth=1)
-
+logoutbutton=Button()
 
 root.mainloop()
\ No newline at end of file
